### ISSUE TITLE
Question: Can someone explain in simple terms what "concurrent auction" means?

### ISSUE THREAD

**Issue by headerbidding** [prebid/Prebid.js#3958]
## Type of issue

Question:
Can someone explain in simpler terms (and with an example) what "concurrent auction" means?

Ref: https://gist.github.com/mkendall07/3dd690e7f624b61cb364fbf8a4ba8dbb

**Comment by jaiminpanchal27** [comment]
@headerbidding 
In 0.x all calls to requestBids were queued so at a time only one auction could run. Subsequent calls to requestBids would have to wait for previous auction to finish. With concurrent auction we removed this and allowed each auction to start as soon as requestBids is executed.

```
pbjs.requestBids(options1) // first auction
pbjs.requestBids(options2) // second auction

In 0.x second auction would be queued if first is not finished.
In >=1.0 both auction will start at the same time in parallel.
```

**Comment by headerbidding** [comment]
Hmmm. I thought there is just one auction for each ad and the highest bidder wins.

**Comment by headerbidding** [comment]
I still dont understand

**Comment by bretg** [comment]
> I thought there is just one auction for each ad and the highest bidder wins.

There is just one auction per adunit. But there are use cases like infinite scroll where there's a series of calls to `pbjs.requestBids()`

1. pbjs.requestBids() - covers the initial adunits on the page
2. pbjs.requestBids() - covers the first 300x250 in a dynamic div on scroll
3. pbjs.requestBids() - covers the second 300x250 in a dynamic div on scroll
4. pbjs.requestBids() - covers the third 300x250 in a dynamic div on scroll

`Concurrent auction` means that subsequent calls to requestBids don't have to wait for their predecessors. So the auction #4 above can start before #3 completes.

**Comment by headerbidding** [comment]
Thank you. Now I got it 👍 -)

**Comment by bretg** [comment]
Cool
